### DISCUSSION TITLE
Timestamps on Players and/or Character Overview

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#331] [Suggestions]
It would be nice to see in our overview, when a player/character was created and when last online maybe?

Just talking about the overview because in the personal character panel you can see that information.

Maybe 2 new tabs in the overview "created" DD/MM/YYYY and "Last online" 1 Hour ago or DD:HH:MM:SS or something else

**Comment by mvaessen** [comment]
This is now available on Admin tools V2
